### ISSUE TITLE
Superadmin container creation in a loop with launch.ps1 (Windows)

### ISSUE THREAD

**Issue by helderbetiol** [ditrit/OGrEE-Core#375]
Even when no superadmin container is created, executing the launch.ps1 script to create it will cause a loop saying the container already exists:
```
Container ogree-superadmin already exists
Container ogree-superadmin-1 already exists
Container ogree-superadmin-2 already exists
Container ogree-superadmin-3 already exists
Container ogree-superadmin-4 already exists
...
```

**Comment by helderbetiol** [comment]
The script executes a `docker ps` command to check with a container with the same name already exists before trying to create it, increment the name if it is the case. The command is not working, it returns `failed to get console mode for stdout: The handle is invalid`. This has been identified as an error from docker version 4.27.0+ when running with Powershell:
https://github.com/docker/for-win/issues/13891
According to the thread, a fix should be released soon with version 4.28.

**Comment by helderbetiol** [comment]
In the meantime, a modification to the script is proposed: if unable to check existing containers, try to create it once with the default name (`ogree-superadmin`). If it does not work because a container already exists with this name, the error will be displayed as follow and no new container will be launched:
```
Unable to check running containers! Try default name only
Launch ogree-superadmin container
docker: Error response from daemon: Conflict. The container name "/ogree-superadmin" is already in use by container "bf4d88f3b016a6c6450e7bc0c66d85431f82d6b97a13f3ca7691b18646202222". You have to remove (or rename) that container to be able to reuse that name.
See 'docker run --help'.
UNABLE TO LAUNCH WEBAPP CONTAINER, CHECK ERROR ABOVE
```
